apps/opik-backend/src/main/java/com/comet/opik/infrastructure/RedisConfig.java

```diff
@@ -19,10 +20,13 @@ public class RedisConfig {
     public Config build() {
         Config config = new Config();
 
+        var redisUrl = RedisUrlParser.parse(singleNodeUrl);
```

Comment:
Variable name is `redisUrl` but it's used for `RedisUrlParser` object, which is misleading. Maybe would be better to rename either variable or record class.